Assets/Mirror/Components/NetworkTransformBase.cs

```diff
@@ -39,6 +40,15 @@ public enum Compression { None, Much, Lots, NoRotation }; // easily understandab
         // This component could be on the player object or any object that has been assigned authority to this client.
         bool isClientWithAuthority => hasAuthority && clientAuthority;
 
+        [Header("Sensitivity")]
+
+        [Tooltip("Changes to the transform must exceed these values to be transmitted on the network.")]
```

Comment:
that's better.
by the way, what problem does this solve?
did anyone mention that they need sensitivity?